rdt-open-reader/build.gradle

```diff
@@ -112,7 +112,7 @@ dependencies {
         exclude group: 'com.android.support', module: 'design'
     }
 
-    implementation('org.smartregister:opensrp-client-core:1.10.8-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-core:1.10.9-SNAPSHOT@aar') {
```

Comment:
I created a release but didn't deploy it to maven, hoping when it's deployed this'll pass tests, https://github.com/OpenSRP/opensrp-client-core/releases

Reply:
Thanks, will rename as `v1.11.0-SNAPSHOT` and deploy.

Reply:
Thanks, will deploy.